metadata_backend/database/db_service.py

```diff
@@ -61,16 +61,20 @@ def __init__(self, database_name: str,
         self.database = db_client[database_name]
 
     @auto_reconnect
-    async def create(self, collection: str, document: Dict) -> bool:
-        """Insert document to collection in database.
+    async def create(self, collection: str, document: Dict,
+                     isFolder: bool = False) -> Union[bool, str]:
+        """Insert document or a folder to collection in database.
 
         :param collection: Collection where document should be inserted
         :param document: Document to be inserted
-        :returns: True if operation was successful
+        :returns: True if operation was successful or ID of the created folder
         """
         result = await self.database[collection].insert_one(document)
         LOG.debug("DB doc inserted.")
-        return result.acknowledged
+        if isFolder:
+            return result.inserted_id
```

Comment:
maybe we can generate something else, to skip relying on internal DB IDs and issues that come with exposing internal ids see `_generate_accession_id` in `operators.py`